.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = C901, E203, E266, E501, E731, W503
+ignore = C901, E203, E231, E266, E501, E731, W503
```

Comment:
Flask / Black spacing and trailing `,`, deactivating in Flask and respecting Black formatting instead